src/agent.js

```diff
@@ -168,6 +169,10 @@ export default class Agent extends WsProto {
     }
   }
 
+  handleTakeOver(S) {
+    alert("You have been logged out by the system. Another user has logged in with the same credentials")
```

Comment:
Please use the same dialog as to display a warning about agent takeover.

Reply:
seems that only alert working in this case